app/res/layout/single_app_view.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical"
+    android:gravity="center" >
+    
+    <TextView 
+        android:id="@+id/app_name"
+        android:textSize="@dimen/font_size_xlargest"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:paddingTop="10sp"
+        android:paddingBottom="5sp"
+        android:textColor="@color/grey_medium"
+        android:gravity="center" />
+    
+    <TextView
+        android:id="@+id/profile_warning"
+        android:textSize="@dimen/font_size_large"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/pre_multiple_apps_warning"
+        android:gravity="center"
+        android:visibility="visible" />
+                    
+    <Button
+         android:id="@+id/uninstall_button"
+         android:text="@string/uninstall_app"
+         android:textSize="@dimen/font_size_xlargest"
+         android:layout_width="fill_parent"
+         android:layout_height="wrap_content"
+         android:onClick="rebootAlertDialog"
+         android:textColor="@color/blue" 
+         android:layout_marginTop="10sp"
+         android:paddingTop="5sp"
+         android:paddingBottom="5sp" />
+                
+    <Button
+         android:id="@+id/archive_button"
+         android:text="@string/archive_app"
+         android:textSize="@dimen/font_size_xlargest"           
+         android:layout_width="fill_parent"
+         android:layout_height="wrap_content"
+         android:onClick="toggleArchived"
+         android:textColor="@color/grey_medium"
+         android:paddingBottom="5sp"
+         android:paddingTop="5sp" />
+    
+    <Button
+        android:id="@+id/update_button"
+        android:text="@string/update_app"
+        android:textSize="@dimen/font_size_xlargest"
+        android:layout_width="fill_parent"
+        android:layout_height="wrap_content"
+        android:onClick="updateClicked"
+        android:textColor="@color/blue"
+        android:paddingBottom="5sp"
+        android:paddingTop="5sp" />
+     
+    <Button
+        android:id="@+id/verify_button"
+        android:text="@string/verify_resources"
+        android:textSize="@dimen/font_size_xlargest"            
+        android:layout_width="fill_parent"
+        android:layout_height="wrap_content"
+        android:clickable="true" 
+        android:onClick="verifyResourcesClicked"
+        android:textColor="@color/grey_medium"
+      	android:paddingBottom="5sp"
+        android:paddingTop="5sp" />
```

Comment:
Mirroring general comment above: I think all margins/etc should be 0, 8, 16, etc padding